clusterloader2/docs/create-new-test.md

```diff
@@ -0,0 +1,67 @@
+-   5000 nodes: [pull-kubernetes-e2e-gce-scale-performance-manual]
+
+Periodics:
+
+-   100 nodes: [ci-kubernetes-e2e-gci-gce-scalability]
+-   5000 nodes: [ci-kubernetes-e2e-gce-scale-performance]
+
+We propose to start with 100 node tests as they are less costly and much
+quicker, shortening debugging cycle and when those tests pass, switch to 5k node
+tests.
+
+## Copy and modify the selected baseline test
+
+Copy the test config to another location within that file. After that it's
+mandatory to edit the following fields:
+
+-   `name` must be unique among all configs.
+-   If testgrid metadata is defined in `annotations`, change it so that the
+    baseline and copied job results are shown in separate views (changing
+    `testgrid-tab-name` should be enough).
+-   If a periodic:
+    -   If the same `gcp-project` or `gcp-project-type` is used then change
+        `cron` so that the baseline and copied jobs won't collide.
+    -   If defined, change perfdash `tags` so that test metrics are shown in
+        separate views.
+    -   In `spec.containers[0].args` set `--cluster` flag to a unique value.
+
+## Adjust the modified baseline test
+
+ClusterLoader2 test configuration consists of a group of templatized YAML files.
+You can find the high level description of the used config language in the
+project [README].
+
+If needed, modify the `CL2_` variables used with specific test in order to tune
+its behaviour. It's always an option to create a new test if needed.
+
+It's also possible to modify values used for other kubekins arguments. For
+reference of their behaviour visit [Kubetest source code].
+
+  [test-infra]: https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-scalability
+  [ProwJobs]: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md
+  [load-test]: https://github.com/kubernetes/perf-tests/tree/master/clusterloader2/testing/load
+  [pull-kubernetes-e2e-gce-100-performance]: https://github.com/kubernetes/test-infra/blob/0ad2c03e7ef5fc974b5209cc7fea1ce1fe8bf4b2/config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml#L5
+  [pull-kubernetes-e2e-gce-scale-performance-manual]: https://github.com/kubernetes/test-infra/blob/0ad2c03e7ef5fc974b5209cc7fea1ce1fe8bf4b2/config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml#L223
+  [ci-kubernetes-e2e-gci-gce-scalability]: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L152
+  [ci-kubernetes-e2e-gce-scale-performance]: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L59
+  [README]: https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/README.md
+  [Kubetest source code]: https://github.com/kubernetes/test-infra/blob/6e76216e7438c6556522fae964e7f050ecb654d1/kubetest/main.go#L134
```

Comment:
It's great there for this link you are using pinned version to specific blob. Could you do this for other links?

Reply:
Done.